src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/DefaultInterfaceImplementationTests.vb

```diff
@@ -3795,10 +3790,36 @@ End Class
 </compilation>
 
             Dim comp1 = CreateCompilation(source1, options:=TestOptions.DebugExe, targetFramework:=TargetFramework.NetStandardLatest, references:={csCompilation})
-            CompileAndVerify(comp1, expectedOutput:=If(ExecutionConditionUtil.IsMonoOrCoreClr, "C.P1.Set", Nothing), verify:=VerifyOnMonoOrCoreClr)
+
+            Dim validator = Sub(m As ModuleSymbol)
+                                Dim p1 = m.GlobalNamespace.GetMember(Of PropertySymbol)("C.P1")
+
+                                Dim i1P1 As PropertySymbol = p1.ExplicitInterfaceImplementations.Single()
+                                Assert.Equal(If(DirectCast(i1P1.ContainingModule, PEModuleSymbol).ImportOptions = MetadataImportOptions.All, "", "WriteOnly ") + "Property I1.P1 As System.Int32", i1P1.ToTestDisplayString())
+
+                                Dim p1Get = p1.GetMethod
+                                Dim p1Set = p1.SetMethod
+
+                                Assert.True(p1Set.IsMetadataVirtual)
+                                Assert.True(p1Set.IsMetadataFinal)
+                                Assert.False(p1Set.IsMustOverride)
+                                Assert.False(p1Set.IsOverridable)
+                                Assert.Equal("Sub I1.set_P1(value As System.Int32)", p1Set.ExplicitInterfaceImplementations.Single().ToTestDisplayString())
+
+                                Assert.False(p1Get.IsMetadataVirtual)
+                                Assert.False(p1Get.IsMetadataFinal)
+                                Assert.False(p1Get.IsMustOverride)
+                                Assert.False(p1Get.IsOverridable)
+                                Assert.Empty(p1Get.ExplicitInterfaceImplementations)
+                            End Sub
+
+            CompileAndVerify(comp1, expectedOutput:=If(ExecutionConditionUtil.IsMonoOrCoreClr, "C.P1.Set", Nothing), verify:=VerifyOnMonoOrCoreClr, sourceSymbolValidator:=validator, symbolValidator:=validator)
+
+            comp1 = CreateCompilation(source1, options:=TestOptions.DebugExe.WithMetadataImportOptions(MetadataImportOptions.All), targetFramework:=TargetFramework.NetStandardLatest, references:={csCompilation})
+            CompileAndVerify(comp1, expectedOutput:=If(ExecutionConditionUtil.IsMonoOrCoreClr, "C.P1.Set", Nothing), verify:=VerifyOnMonoOrCoreClr, sourceSymbolValidator:=validator, symbolValidator:=validator)
```

Comment:
>Nothing [](start = 107, length = 7)

Running this executable on .NET Framework is an unsupported scenario, right? This is just verifying the runtime behavior?

Reply:
> Running this executable on .NET Framework is an unsupported scenario, right? This is just verifying the runtime behavior?

Correct, ```Nothing``` ensures we verify execution only on CoreClr or Mono.

---
In reply to: [313599079](https://github.com/dotnet/roslyn/pull/37914#discussion_r313599079) [](ancestors = 313599079)